### ISSUE TITLE
freeze when I use testResultsProcessor option

### ISSUE THREAD

**Issue by SerzN1** [rogeliog/jest-runner-eslint#7]
Hi guys. Thanks for runner!
What's wrong with my config? Jest just freeze in watch mode after single run.
I have junit reporter installed in my `package.json`
```"testResultsProcessor": "./node_modules/jest-junit-reporter",```

**Comment by rogeliog** [comment]
Hi 👋thanks for reporting!

Does it work when you remove the reporter?

**Comment by SerzN1** [comment]
Yep. May be it needs some improvements to use this reporter or may be reporter should be modified?
